src/models.d.ts

```diff
@@ -976,12 +1019,560 @@ export namespace Models {
+         */
+        error: string;
+        /**
+         * Is column required?
+         */
+        required: boolean;
+        /**
+         * Is column an array?
+         */
+        array?: boolean;
+        /**
+         * Column creation date in ISO 8601 format.
+         */
+        $createdAt: string;
+        /**
+         * Column update date in ISO 8601 format.
+         */
+        $updatedAt: string;
+        /**
+         * String format.
+         */
+        format: string;
+        /**
+         * Default value for column when not provided. Cannot be set when column is required.
+         */
+        xdefault?: string;
+    }
+    /**
+     * ColumnURL
+     */
+    export type ColumnUrl = { 
+        /**
+         * Column Key.
+         */
+        key: string;
+        /**
+         * Column type.
+         */
+        type: string;
+        /**
+         * Column status. Possible values: `available`, `processing`, `deleting`, `stuck`, or `failed`
+         */
+        status: string;
+        /**
+         * Error message. Displays error generated on failure of creating or deleting an column.
+         */
+        error: string;
+        /**
+         * Is column required?
+         */
+        required: boolean;
+        /**
+         * Is column an array?
+         */
+        array?: boolean;
+        /**
+         * Column creation date in ISO 8601 format.
+         */
+        $createdAt: string;
+        /**
+         * Column update date in ISO 8601 format.
+         */
+        $updatedAt: string;
+        /**
+         * String format.
+         */
+        format: string;
+        /**
+         * Default value for column when not provided. Cannot be set when column is required.
+         */
+        xdefault?: string;
+    }
+    /**
+     * ColumnDatetime
+     */
+    export type ColumnDatetime = { 
+        /**
+         * Column Key.
+         */
+        key: string;
+        /**
+         * Column type.
+         */
+        type: string;
+        /**
+         * Column status. Possible values: `available`, `processing`, `deleting`, `stuck`, or `failed`
+         */
+        status: string;
+        /**
+         * Error message. Displays error generated on failure of creating or deleting an column.
+         */
+        error: string;
+        /**
+         * Is column required?
+         */
+        required: boolean;
+        /**
+         * Is column an array?
+         */
+        array?: boolean;
+        /**
+         * Column creation date in ISO 8601 format.
+         */
+        $createdAt: string;
+        /**
+         * Column update date in ISO 8601 format.
+         */
+        $updatedAt: string;
+        /**
+         * ISO 8601 format.
+         */
+        format: string;
+        /**
+         * Default value for column when not provided. Only null is optional
+         */
+        xdefault?: string;
```

Comment:
I think I saw in Node.js SDK that xdefault is just default. Is this Deno specific problem that it cannot use word "default" here?

Or just developer experience improvement? If so, why not in Node?